tools/combine_metaphlan_humann/combine_metaphlan_humann.xml

```diff
@@ -0,0 +1,112 @@
+<tool id="combine_metaphlan_humann" name="Combine MetaPhlAn and HUMAnN outputs" version="0.3.0" profile="22.01">
+    <description>to relate genus/species abundances and gene families/pathways abundances</description>
+    <requirements>
+    </requirements>
+    <stdio>
```

Comment:
stdio can be removed in this form and replaced with "detect_error" on the command tag

Reply:
Done